scripts/pg2sphinx_trigger.py

```diff
@@ -1,25 +1,26 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 #
-# read sphinx config file and update all indexes related to input database or input table or index pattern
+# read sphinx config file and update all indexes related to input database or
+# input table or index pattern
 
 import os
-import sys
-import time
-import getpass
 import re
 import sqlite3
-import psycopg2
-import optparse
 import subprocess
-from optparse import OptionParser
+import sys
+from optparse import OptionParser  # pylint: disable=deprecated-module
```

Comment:
Why not yet use `argparse` to avoid this warning. This module might be soon not be anymore supported and we will need then to do the changes anyway.